release-notes/major9/09.19.0/valtimo-frontend-libraries.md

```diff
@@ -24,6 +24,31 @@ The following features were added:
 
   The `valtimo-modal` component now scrolls to the top when transitioning between form flow steps, and when switching pages in a Form.IO wizard.
 
+* **Dossier detail tab with zaakobjecten**
+
+  Added a tab page with a dropdown to select an object type which is linked to the current case. After selecting this
```

Comment:
A suggestion, but I'm not completely sure about it (because i'm not native english speaker) but I believe that in this case '**that is**' would fit better instead of '**which is**'.

_Added a tab page with a dropdown to select an object type_ `that is` _linked to the current case._ 

overall, I think it's clear and well descriptive how to use